README.md

```diff
@@ -219,6 +220,17 @@ These are features that Dolibarr does **not** yet fully support:
 - Payroll module
 - No native embedded Webmail, but you can send email to contacts in Dolibarr with e.g. offers, invoices, etc.
 - Dolibarr can't do coffee (yet)
+- The REST API currently cannot:
+  - Update a supplier order 
+  - Get tickets
```

Comment:
Seems available in develop branch

Reply:
Correct! https://github.com/Dolibarr/dolibarr/blob/develop/htdocs/ticket/class/api_tickets.class.php
I'll remove it :)